.github/ISSUE_TEMPLATE/spike.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Spike
+about: Research and/or Analysis Required
+title: ''
+labels: ":zap: spike"
+assignees: ''
+
+---
+
+# Why: Problem description :grey_question:
+A clear and concise description of what the problem is. 
+- [ ] Describe the matter that needs investigating
+- [ ] Provide context for posterity
+
+# What: Describe what a solution should resolve :grey_exclamation:  
+A clear and concise description of what a working solution should resolve, and any required deliverables (i.e. issue cards, prototypes, POCs etc.)
+
+# Describe any ideas you've considered :bulb:
+A clear and concise description of any alternative solutions or features you've considered.
+Implementation suggestions or options may be provided.
+
+**Proposed timebox**
+(Optional) in days
+
+**Additional context**
+Add any other context or screenshots about the feature request here: notion doc, github discussion or ticket
+
+# Project management
+- [ ] I have created any required issues or epics
+- [ ] I have updated the related notion or epic documentation
```

Comment:
Are these intended to be checked off as the spike is being completed? If so, it might be worth adding a note to clarify this -- another interpretation is "you should check these off as you are filling out this template".

Reply:
It was meant to mimic your pull request template, like a reminder to update issues / epic / notion documentation.

Reply:
Right. My question is:

1. Am I intended to check them off _before submitting the issue_?
2. Or am I intended to check them off _as I "execute" the spike_, to indicate that I'm done with it?

I think it's (2), so maybe this merits a bit of clarification, such as:
```suggestion
Before closing this card,
- [ ] I have created any required issues or epics
- [ ] I have updated the related notion or epic documentation
```

Reply:
Sounds good!